### ISSUE TITLE
[TreeView] Implement property defaultExpandedChildren on v1 component

### ISSUE THREAD

**Issue by David74000** [pnp/sp-dev-fx-controls-react#808]
#### Category
[ ] Enhancement

[ ] Bug

[x] Question

#### Version

1.2

#### Expected / Desired Behavior / Question

Hello, I am using the TreeView component in SP2019 on-premise (version 1.2 of spfx-controls-react). I would like to implement the behaviour allowed by the property **defaultExpandedChildren** but this property is only available in version 2.1 of the spfx-controls-react. I would like to know if it feasable to apply the code used for the property defaultExpandedChildren into my old version ?
I assume I need to modify the JS and TS files used in the node_modules folder but I first would like to know your opinion on that ?
Cheers
Thanks!

David

**Comment by AJIXuMuK** [comment]
Hello @David74000 ,

You can `pull` this repo to your machine and copy the `TreeView` control completely.

I marked this issue for `v1`. But I don't have any dates or promises regarding v1 version.

**Comment by David74000** [comment]
Hi @AJIXuMuK 
thank you for your feedback. You mean pull the repo, copy the TreeView component (+ any dependency) and integrate it as a new component in my solution ? Like renaming it to TreeViewCustom to avoid conflict with other pnp-react component ?
Cheers
David

**Comment by AJIXuMuK** [comment]
@David74000 yes. You don’t even need to rename it if you don’t `import` the `TreeView` from the library.

**Comment by David74000** [comment]
will try it, thank you again !

**Comment by michaelmaillot** [comment]
Hi @David74000,

Did you figure out a solution for your context?
